packages/SidePanel/src/hooks/useFooterOffset.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import debounce from "lodash.debounce";
+
+const useFooterOffset = (height, ref = null, isSticky) => {
```

Comment:
This approach seems way more complicated than simply using a different (flexbox) layout when the `isSticky` prop is true.  No handlers, no debouncing... just flex-shrink, flex-grow, and overflow.

Reply:
Also... it still looks janky.  It floats and then reappears.  😕

Reply:
I added a comment about that the footer needs an upgrade, I added it as the last feature, this will not affect the API and is easier to refactor it in a new PR. 

I don't think we should stop the whole SidePanel only for the footer.

Reply:
I created an issue for this already https://github.com/acl-services/paprika/issues/97

Reply:
Okay, I agree.

Reply:
Huh... I thought there was a specific reason we're not using flexbox here.